Simplified/NYPLBookRegistry.h

```diff
@@ -123,7 +123,7 @@ genericBookmarks:(nullable NSArray<NYPLBookLocation *> *)genericBookmarks;
 // This method should be called whenever new book information is retrieved from a server, but may
 // not include user-specific information. We want to update the metadata, but not overwrite the
 // existing availability information and acquisition URLs.
-- (void)updateBookMetadata:(nonnull NYPLBook *)book;
+- (NYPLBook *)updatedBookMetadata:(NYPLBook *)book;
```

Comment:
can you update the documentation with the return value info?

Reply:
Done